Tasks/Andrey Kruk/task_2/task2.py

```diff
@@ -0,0 +1,64 @@
+import datetime
+
+
+hours = {1: 'один час', 2: 'два часа', 3: 'три часа', 4: 'четыре часа', 5: 'пять часов', 6: 'шесть часов',
+         7: 'семь часов', 8: 'восемь часов', 9: 'девять часов', 10: 'десять часов', 11: 'одиннадцать часов',
+         12: 'двенадцать часов'}
+hours_case = {1: 'первого', 2: 'второго', 3: 'третьего', 4: 'четвертого', 5: 'пятого', 6: 'шестого',
+              7: 'седьмого', 8: 'восьмого', 9: 'девятого', 10: 'десятого', 11: 'одиннадцатого',
+              12: 'двенадцатого'}
+minutes = {1: 'одна минута', 2: 'две минуты', 3: 'три минуты', 4: 'четыре минуты', 5: 'пять минут',
+           6: 'шесть минут', 7: 'семь минут', 8: 'восемь минут', 9: 'девять минут', 10: 'десять минут',
+           11: 'одиннадцать минут', 12: 'двенадцать минут', 13: 'тринадцать минут', 14: 'четырнадцать минут',
+           15: 'пятнадцать минут', 16: 'шестнадцать минут', 17: 'семнадцать минут', 18: 'восемнадцать минут',
+           19: 'девятнадцать минут', 20: 'двадцать минут', 21: 'двадцать одна минута',
+           22: 'двадцать две минуты', 23: 'двадцать три минуты', 24: 'двадцать четыре минуты',
+           25: 'двадцать пять минут', 26: 'двадцать шесть минут', 27: 'двадцать семь минут',
+           28: 'двадцать восемь минут', 29: 'двадцать девять минут', 31: 'дридцать одна минута',
+           32: 'тридцать две минуты', 33: 'тридцать три минуты', 34: 'тридцать четыре минуты',
+           35: 'тридцать пять минут', 36: 'тридцать шесть минут', 37: 'тридцать семь минут',
+           38: 'тридцать восемь минут', 39: 'тридцать девять минут', 40: 'сорок минут', 41: 'сорок одна минута',
+           42: 'сорок две минуты', 43: 'сорок три минуты', 44: 'сорок четыре минуты'}
+minutes_remain = {45: 'пятнадцати минут', 46: 'четырнадцати минут', 47: 'тринадцати минут', 48: 'двенадцати минут',
+                  49: 'одиннадцати минут', 50: 'десяти минут', 51: 'девяти минут', 52: 'восьми минут',
+                  53: 'семи минут', 54: 'шести минут', 55: 'пяти минут', 56: 'четырех минут',
+                  57: 'трех минут', 58: 'двух минут', 59: 'одной минуты'}
```

Comment:
I would suggest to store only numerals in these dictionaries, and to format correct "minutes" token in the main logic